frontend/server/src/DAO/Contests.php

```diff
@@ -500,14 +501,14 @@ final public static function getContestsParticipating(
         $params[] = intval($offset);
         $params[] = intval($pageSize);
 
-        /** @var list<array{admission_mode: string, alias: string, contest_id: int, contestants: int, description: string, finish_time: \OmegaUp\Timestamp, last_updated: \OmegaUp\Timestamp, organizer: string, original_finish_time: \OmegaUp\Timestamp, partial_score: bool, problemset_id: int, recommended: bool, rerun_id: int, scoreboard_url: string, scoreboard_url_admin: string, start_time: \OmegaUp\Timestamp, title: string, window_length: int|null}> */
+        /** @var list<array{admission_mode: string, alias: string, contest_id: int, contestants: int, description: string, finish_time: \OmegaUp\Timestamp, last_updated: \OmegaUp\Timestamp, organizer: string, original_finish_time: \OmegaUp\Timestamp, partial_score: bool, participating: bool, problemset_id: int, recommended: bool, rerun_id: int, scoreboard_url: string, scoreboard_url_admin: string, start_time: \OmegaUp\Timestamp, title: string, window_length: int|null}> */
         return \OmegaUp\MySQLConnection::getInstance()->GetAll($sql, $params);
```

Comment:
Ahh, este problema con los campos de tipo bool ya nos ha ocurrido en ocasiones anteriores. 

Lo que se me ocurre para solucionarlo es hacer el cast desde PHP, algo así como esto:
```suggestion
        /** @var list<array{admission_mode: string, alias: string, contest_id: int, contestants: int, description: string, finish_time: \OmegaUp\Timestamp, last_updated: \OmegaUp\Timestamp, organizer: string, original_finish_time: \OmegaUp\Timestamp, partial_score: bool, participating: int, problemset_id: int, recommended: bool, rerun_id: int, scoreboard_url: string, scoreboard_url_admin: string, start_time: \OmegaUp\Timestamp, title: string, window_length: int|null}> */
        $rs = \OmegaUp\MySQLConnection::getInstance()->GetAll($sql, $params);

        $contests = [];
        foreach ($rs as $row) {
            $row['participating'] = boolval($row['participating']);
            $contests[] = $row;
        }
        return $contests;
```

Reply:
@lhchavez es válido?

Reply:
sí, MySQL no tiene manera de hacer cast a booleanos :/ las expresiones "booleanas" (que no vienen directamente de una columna declarada explícitamente como `TINYINT`) normalmente salen como enteros normales.